### ISSUE TITLE
Terminal not working

### ISSUE THREAD

**Issue by nuxes** [steamos-community/stephensons-rocket#105]
In desktop mode, I click the icon for Terminal, a tab opens at the top and I get the spinning wheel for a few seconds, then they disappear.  Running this in VMware.
![capture](https://cloud.githubusercontent.com/assets/13204390/9422751/2bac8114-4868-11e5-9a90-be8751f3b3ff.JPG)

**Comment by sharkwouter** [comment]
does xterm work?

**Comment by directhex** [comment]
https://github.com/ValveSoftware/SteamOS/issues/321 ?

**Comment by nuxes** [comment]
How do I get to xterm?

Sounds similar to that issue.

**Comment by Ryochan7** [comment]
Alt+F2 brings up a run dialog. You can run xterm from that.

**Comment by nuxes** [comment]
Thanks, that works.
I guess the terminal is a known bug with vanilla SteamOS?

**Comment by mdeguzis** [comment]
Yes. For now, you can use the fix described in the issue directhex linked. It is the same issue.

**Comment by sharkwouter** [comment]
Today I've seen this for the first time, maybe it had something to do with doing a custom install.

**Comment by directhex** [comment]
It's an issue if you install with a non-English locale, afaik

**Comment by mdeguzis** [comment]
Happens to me. I'll have to retest again tonight if I can. If not this weekend.

**Comment by sharkwouter** [comment]
I had it on a custom installation on which I selected the US as location, English as language and the US International keyboard layout.
